### ISSUE TITLE
xmr-stak Win64 freezes randomly since March 11

### ISSUE THREAD

**Issue by Swericor** [fireice-uk/xmr-stak-amd#212]
I have four miners running xmr-stak. Three running Windows 10 and one Ubuntu. From earlier today (March 11, 2018) the xmr-stak on the three Windows machines freezes randomly. I have diasbled Quick Edit mode in CMD and I mine on different pools. I have not experienced this at all before, and now - at the same time - all my xmr-stak installations in Windows are unstable, while the Ubuntu rig runs as before.
I noticed that one machine froze when switching to dev pool.  

All help appreciated!

**Comment by psychocrypt** [comment]
Check if windows got an auto update.

**Comment by Swericor** [comment]
No common ones for the three machines. One installed KB4058043 and KB4074588 earlier today. The other ones have them from before. No Defender updates either.

**Comment by psychocrypt** [comment]
Than check what both updates do. I think one of these is the source of your
issues.

**Comment by Swericor** [comment]
Ok, here is an update on this issue. I have identified the problem and it was not related to xmr-stak (yes, this post is misplaced and should have been under the xmr-stak and not xmr-stak-amd branch). When the rigs auto-restarted after the Windows update the OpenCL device order was changed. Since I have different AMD GPUs in the rigs the amd.txt config became invalid for the setups, making the rigs very unstable. I didn't have any screens nor HDMI dummies connected to the rigs at the time. A simple reboot and adding one HDMI dummy per rig fixed the OpenCL device order issue and since then all rigs are stable.
